### ISSUE TITLE
Add a possibility to get reactive limits of groups

### ISSUE THREAD

**Issue by vincentbarbesant** [powsybl/pypowsybl#197]
* **Do you want to request a *feature* or report a *bug*?**
This is a feature request.

* **What is the current behavior?**
The returned dataframe for group does not contain any information about reactive limits of groups :
```
Data columns (total 12 columns):
 #   Column                Non-Null Count  Dtype  
---  ------                --------------  -----  
 0   energy_source         2768 non-null   object 
 1   target_p              2768 non-null   float64
 2   max_p                 2768 non-null   float64
 3   min_p                 2768 non-null   float64
 4   target_v              2768 non-null   float64
 5   target_q              2768 non-null   float64
 6   voltage_regulator_on  2768 non-null   bool   
 7   p                     2768 non-null   float64
 8   q                     1435 non-null   float64
 9   voltage_level_id      2768 non-null   object 
 10  bus_id                2768 non-null   object 
 11  genreCvg              2768 non-null   object 
```

* **What is the expected behavior?**
MVP : I would like to be able to get the reactive limits of the group at current active power in the group datraframe
Nice to have : I would like to be able to get the reactive limits of the group for a given active power

* **What is the motivation / use case for changing the behavior?**
For voltage control, I need to know if a group has some margin left in reactive power

* **Please tell us about your environment:**
  - PyPowsybl Version : 0.8.0
  - OS Version: Windows 10 Entreprise 64-bit (10.0, Build 18363) (18362.19h1_release.190318-1202)

**Comment by geofjamg** [comment]
Done in #129 and will be available in next release (0.9)

**Comment by vincentbarbesant** [comment]
Hello,

Using the 0.9.0 version, I got some NaNs in `min_q` and `max_q` columns of `get_generators()`, although I have some values when I call `get_reactive_capability_curve_points()` : 

```
                 energy_source     target_p         min_p         max_p  min_q  max_q    target_v    target_q  voltage_regulator_on regulating_terminal_voltagelevel            p           q            i voltage_level_id     bus_id  connected        genreCvg
id                                                                                                                                                                                                                                                                
ARGOEIN1                   WIND     1.543953      0.000000     17.500000    NaN    NaN  238.650467    0.000000                 False                          ARGOEP6    -1.543953   -0.000000     3.698560          ARGOEP6  ARGOEP6_0       True          EOLIEN
EGUZO4EGUZOH1+2+3         HYDRO     0.000000      0.000000     35.279999    NaN    NaN   95.170685   -0.000000                  True                          EGUZOP4    -0.000000  -11.004186    66.756581          EGUZOP4  EGUZOP4_0       True       TURBINAGE
EGUZO4EGUZOH4+5+6         HYDRO     0.000000      0.000000     37.259998    NaN    NaN   94.479713   -0.000000                  True                          EGUZOP4    -0.000000  -20.000000   122.646858          EGUZOP4  EGUZOP4_3       True       TURBINAGE
HOURA.HG1                 HYDRO     0.000000      0.000000     22.500000    NaN    NaN    0.000000    0.000000                 False                          HOURAP1    -0.000000         NaN          NaN          HOURAP1                 False       TURBINAGE
HOURA.HG2                 HYDRO     0.000000      0.000000     22.500000    NaN    NaN    0.000000    0.000000                 False                          HOURAP1    -0.000000         NaN          NaN          HOURAP1                 False       TURBINAGE
MIEGE1G1                  HYDRO     0.000000      0.000000     23.000000    NaN    NaN    0.000000    0.000000                 False                          MIEGEP1    -0.000000         NaN          NaN          MIEGEP1                 False       TURBINAGE
MIEGE1G2                  HYDRO     0.000000      0.000000     27.000000    NaN    NaN    0.000000    0.000000                 False                          MIEGEP1    -0.000000         NaN          NaN          MIEGEP1                 False       TURBINAGE
MIEGE1G3                  HYDRO     0.000000      0.000000     27.000000    NaN    NaN    0.000000    0.000000                 False                          MIEGEP1    -0.000000         NaN          NaN          MIEGEP1                 False       TURBINAGE
RUEYRIN1                  SOLAR     0.000000      0.000000     12.600000    NaN    NaN    0.000000    0.000000                 False                          RUEYRP3    -0.000000         NaN          NaN          RUEYRP3                 False  PHOTOVOLTAIQUE
SSDAL1GROUPE2             HYDRO    10.025670      0.000000     12.500000    NaN    NaN   20.416990   -0.284622                  True                          SSDALP1   -10.080204   -0.993236   286.427720          SSDALP1  SSDALP1_0       True       TURBINAGE
SSDAL1GROUPE3             HYDRO     0.000000      0.000000     12.500000    NaN    NaN   20.416990   -0.284622                  True                          SSDALP1    -0.000000   -0.993236    28.086661          SSDALP1  SSDALP1_0       True       TURBINAGE
SSDAL3GROUPE1             HYDRO     0.000000      0.000000     12.500000    NaN    NaN    0.000000    0.000000                 False                          SSDALP3    -0.000000         NaN          NaN          SSDALP3                 False       TURBINAGE
SSDAL3GROUPE4             HYDRO     0.000000      0.000000     12.500000    NaN    NaN    0.000000    0.000000                 False                          SSDALP3    -0.000000         NaN          NaN          SSDALP3                 False       TURBINAGE
VIELMIN1                   WIND     5.002809      0.000000     30.000000    NaN    NaN   65.974976    0.000000                 False                          VIELMP3    -5.002809   -0.000000    43.842205          VIELMP3  VIELMP3_0       True          EOLIEN
VIELMIN2                   WIND     6.561801      0.000000     32.000000    NaN    NaN   65.974976    0.000000                 False                          VIELMP3    -6.561801   -0.000000    57.504457          VIELMP3  VIELMP3_0       True          EOLIEN
VIELMIN3                   WIND     5.108079      0.000000     21.600000    NaN    NaN   65.974976    0.000000                 False                          VIELMP3    -5.108079   -0.000000    44.764739          VIELMP3  VIELMP3_0       True          EOLIEN
BREUI6AIGLEG1             HYDRO     0.000000      0.000000     49.500000    NaN    NaN  238.345612  -29.643250                  True                          BREUIP6    -0.000000         NaN          NaN          BREUIP6                 False       TURBINAGE
BREUI6AIGLEG2             HYDRO     0.000000      0.000000     49.500000    NaN    NaN  237.672775  -23.291124                  True                          BREUIP6    -0.000000         NaN          NaN          BREUIP6                 False       TURBINAGE
BREUI6AIGLEG3             HYDRO     0.000000      0.000000     49.500000    NaN    NaN  237.672775  -27.525875                  True                          BREUIP6    -0.000000         NaN          NaN          BREUIP6                 False       TURBINAGE
BREUI6AIGLEG4             HYDRO     0.000000      0.000000     49.500000    NaN    NaN  238.345612  -29.643250                  True                          BREUIP6    -0.000000         NaN          NaN          BREUIP6                 False       TURBINAGE
BREUI6AIGL8G6             HYDRO     0.000000     86.000000    139.199997    NaN    NaN    0.000000    0.000000                 False                          BREUIP6    -0.000000         NaN          NaN          BREUIP6                 False       TURBINAGE

```

Would it be possible to put the `min_q`and `max_q` at current power in the generators dataframe ?
